codec/src/test/java/io/netty/handler/codec/MessageAggregatorTest.java

```diff
@@ -78,9 +78,10 @@ public void testReadFlowManagement() throws Exception {
 
         assertFalse(embedded.writeInbound(first));
         assertFalse(embedded.writeInbound(chunk));
-        assertTrue(embedded.writeInbound(last));
+        assertFalse(embedded.writeInbound(last));
 
-        assertEquals(3, counter.value); // 2 reads issued from MockMessageAggregator
+        embedded.read();
+        assertEquals(2, counter.value); // 2 reads issued from MockMessageAggregator
```

Comment:
@carryxyh can you explain why these changes are needed ?

Reply:
first time increase was in EmbeddedChannel's contructor.
second time was `embedded.read()`.